src/rgw/rgw_process.cc

```diff
@@ -231,7 +231,12 @@ int process_request(RGWRados* const store,
 
   dout(1) << "====== req done req=" << hex << req << dec
 	  << " op status=" << op_ret
+	  << " op_name=" << (op ? op->name() : "unknown")
+	  << " user_id=" << (s->user ? s->user->user_id.to_str() : "nobody")
```

Comment:
Can we replace  `"nobody"`  with  `""` ?  Because `nobody` is a valid uid, which can be created by user with radsgw-admin cmd.